### PR TITLE
HACKATHON: Extend context around redshift snapshot policy

### PR DESCRIPTION
**Pull request by djdoeslinux**

### CODE REVIEW

File: services/config.rb [hunk 1]
```diff
@@ -235,6 +235,13 @@
     query(func: uid(clusters)) @filter( le(val(retention_period), 10) ) {
       <%= default_predicates %>
       automated_snapshot_retention_period
+      relates_to @filter(has(snapshot)) {
+        <%= default_predicates %>
+        cluster_identifier
+	snapshot_create_time
+        status
+        encrypted
+      }
```
Comment (armaneous) [inline services/config.rb]: Probably good to also include relationships to `vpc` (two relationships away from `db_instance`) and things like `db_parameter_group` or `db_subnet_group` (and probably others).

**Comment by scvenema** [comment]
Just pushed an update for using the new visualization query format

File: services/config.rb [hunk 2]
```diff
@@ -122,12 +118,10 @@
     clusters as var(func: <%= filter['cluster'] %>) {
       is_version_upgrade_allowed as allow_version_upgrade
     }
-
     no_upgrade_allowed_clusters as query(func: uid(clusters)) @filter(eq(val(is_version_upgrade_allowed), false)) {
       <%= default_predicates %>
       is_version_upgrade_allowed
     }
-
     vizualize(func: uid(no_upgrade_allowed_clusters)) {
```
Comment (armaneous) [inline services/config.rb]: This is no longer needed, should add `meta_rule_visualize` like below rule.

File: services/config.rb [hunk 3]
```diff
@@ -122,12 +118,10 @@
     clusters as var(func: <%= filter['cluster'] %>) {
       is_version_upgrade_allowed as allow_version_upgrade
     }
-
     no_upgrade_allowed_clusters as query(func: uid(clusters)) @filter(eq(val(is_version_upgrade_allowed), false)) {
```
Comment (armaneous) [inline services/config.rb]: No need to alias since alias will go unused

File: services/config.rb [hunk 4]
```diff
@@ -271,9 +265,24 @@
     clusters as var(func: <%= filter['cluster'] %>) {
       retention_period as automated_snapshot_retention_period
     }
-    query(func: uid(clusters)) @filter( le(val(retention_period), 10) ) {
+    snapshot_retention as query(func: uid(clusters)) @filter( le(val(retention_period), 10) ) {
```
Comment (armaneous) [inline services/config.rb]: Alias now unnecessary, will go unused and error.

**Pull request merged**